Makefile

```diff
@@ -192,6 +192,7 @@ endif
 	@echo "Getting locust master log:"
 	kubectl logs --namespace $(LOCUST_NAMESPACE) -f -l performance-test-pod-name=$(SCENARIO)-test-master | tee load-test.log
 	date --utc -Ins>$(TMP_DIR)/benchmark-after
+	oc exec rhdh-postgresql-primary-0 -n rhdh-performance -- sh -c 'cat  /var/lib/pgsql/data/userdata/log/postgresql*.log' | grep duration > postgresql.log
```

Comment:
@yogananth-subramanian when installing RHDH with OLM, the name of the DB pod is different - `backstage-psql-developer-hub-0`, please take that into account to support both Helm and OLM